styles/sections/_mission.scss

```diff
@@ -0,0 +1,106 @@
+@use '../reset';
```

Comment:
Do we need to use the reset styles here? I thought it was fine to only add it to the top of our `index` styles, but let me know if i'm missing something :-)

Reply:
No you're right--removed